### PR TITLE
Add Media CDN security policy sample

### PR DESCRIPTION
**Pull request by msampathkumar**

### CODE REVIEW

File: media_cdn_security_policies/test.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 Google LLC
```
Comment (betsy-lichtenberg) [inline media_cdn_security_policies/test.yaml:1]: Why are you skipping testing?

File: media_cdn_security_policies/test.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: blueprints.cloud.google.com/v1alpha1
+kind: BlueprintTest
+metadata:
+  name: media_cdn_security_policies
+spec:
+#  skip: true
```
Comment (betsy-lichtenberg) [inline media_cdn_security_policies/test.yaml:20]: Instead of commenting out, maybe just delete the file. Having no test.yaml file is the standard.

**Comment by comment-bot-dev** [comment]
@msampathkumar
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_headers
 All files need a license header. Please make sure all your files include the appropriate header. A helper tool available [here](https://github.com/google/addlicense). 
 ```bash 
Checking file headers
./media_cdn_security_policies/main.tf
 ```

**Comment by comment-bot-dev** [comment]
@msampathkumar
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_headers
 All files need a license header. Please make sure all your files include the appropriate header. A helper tool available [here](https://github.com/google/addlicense). 
 ```bash 
Checking file headers
./media_cdn_security_policies/main.tf
 ```

**Comment by comment-bot-dev** [comment]
@msampathkumar
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_headers
 All files need a license header. Please make sure all your files include the appropriate header. A helper tool available [here](https://github.com/google/addlicense). 
 ```bash 
Checking file headers
./media_cdn_security_policies/main.tf
 ```

**Comment by comment-bot-dev** [comment]
@msampathkumar
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Review by betsy-lichtenberg — APPROVED** [review]

**Pull request merged**